dGame/dGameMessages/GameMessages.h

```diff
@@ -373,6 +374,15 @@ namespace GameMessages {
 
 	void SendDisplayChatBubble(LWOOBJID objectId, const std::u16string& text, const SystemAddress& sysAddr);
 
+	/**
+	 * @brief
+	 *
+	 * @param objectId ID of the entity to set idle flags
+	 * @param FlagsOn Flag to turn on
+	 * @param FlagsOff Flag to turn off
+	 */
+	void SendChangeIdleFlags(LWOOBJID objectId, eAnimationFlags FlagsOn, eAnimationFlags FlagsOff, const SystemAddress& sysAddr);
```

Comment:
These default to zero for what its worth per [the struct](https://lcdruniverse.org/lu_packets/src/lu_packets/world/gm/client/mod.rs.html#1363-1368)

Reply:
Ty will make it default to 0